app/views/organisers/ongoing_projects.slim

```diff
@@ -23,5 +23,6 @@
                  = link_to project.mentor.full_name, mentor_profile_path(project.mentor.id)
               td = project.title
               td.text-center
+                - if project.github_link.present?
```

Comment:
hey, I made that because I though it will be easier to validate which projects actually have github url set. Sorry that it's in the same pull request as this other feature. 

Tell me if it's not ok for you